requirements.txt

```diff
@@ -9,7 +9,7 @@
 #
 # In case you want to test a patch with your own CCM branch, further to changing below CCM repo and branch name, you need to add -e flag at the beginning
 # Example: -e git+https://github.com/userb/ccm.git@cassandra-17182#egg=ccm
-git+https://github.com/apache/cassandra-ccm.git@cassandra-test#egg=ccm
+-e git+https://github.com/dkropachev/cassandra-ccm.git@dk/move-off-distutils-version#egg=ccm
```

Comment:
I understand that you want cicd to pass and merge it, but wouldn't it better to point it to a branch from `git+https://github.com/apache/cassandra-ccm.git` ?